packages/odata-service-writer/test/index.test.ts

```diff
@@ -38,47 +37,37 @@ describe('ODataService templates', () => {
         }
     });
 
-    it('generate: invalid project with faulty manifest.json', async () => {
-        const testDir = join(outputDir, 'invalid-project');
-
-        const ui5Yaml = (await UI5Config.newInstance('')).addFioriToolsProxydMiddleware({ ui5: {} }).toString();
-        fs.write(join(testDir, 'ui5.yaml'), ui5Yaml);
-        fs.write(join(testDir, 'ui5-local.yaml'), '');
-        fs.writeJSON(join(testDir, 'package.json'), { ui5: { dependencies: [] } });
-        const manifestPath = join(testDir, 'webapp/manifest.json');
-        fs.write(manifestPath, '{}');
-
-        await expect(generate(testDir, validServiceConfig, fs)).rejects.toEqual(
-            Error(t('error.requiredProjectPropertyNotFound', { property: `'sap.app'.id`, path: manifestPath }))
-        );
+    beforeEach(() => {
+        fs = create(createStorage());
     });
 
-    it('generate: valid project with standard valid input', async () => {
-        const testDir = join(outputDir, 'odata-service-v2');
+    async function createTestDir(name: string): Promise<string> {
```

Comment:
Does this directory need to be (optionally) cleaned up after a successful test run?

Reply:
`fs` is a reference to the `mem-fs-editor` here, therefore, nothing is written to the real filesystem. Additionally, `beforeEach` run, we create a new reference to `fs`, so each test start with a clean directory.